src/pint/models/timing_model.py

```diff
@@ -1502,6 +1496,57 @@ def delete_jump_and_flags(self, toa_table, jump_num):
             return
         self.components["PhaseJump"].setup()
 
+    def get_spin_freq(self, times, calctype="modelF0"):
```

Comment:
Should this allow `times=None` when just using `modelF0` to avoid forcing users to make up a bogus time?

Reply:
That's not a bad idea.  Although if you know you are not giving times *and* explicitly specifying "modelF0", why wouldn't you just use `m.F0.quantity` rather than calling a method?